enhancements/network/service-egress-traffic-steering.md

```diff
@@ -159,32 +169,32 @@ the egress traffic flow for the pod `10.244.1.6` on `ovn-worker2` towards an ext
 ```
 As mentioned earlier, for the opposite direction (ingress/external client initiates) to work properly the LoadBalancer provider needs to announce the service only from `ovn-worker`.
 
+When the `snat` field is not specified, `ovnkube-master` does not need to create any logical router policies as the egress packets of each pod would exit through the pod's node.
+However, `ovnkube-master` will set the status field of the resource with `host: ALL` to designate that no reroute logical router policies exist for the service, "instructing" all of the `ovnkube-nodes` to handle the resource's `Network` field without creating SNAT iptables rules.
+
 #### Network
-In addition, an `EgressService` supports a `network` field.
-When it is specified the relevant `ovnkube-node` takes care of creating an additional ip rule and iptables rules on its host.
-Assuming an `EgressService` has `Network: blue`, a ClusterIP of 10.96.135.5 and its endpoints are 10.244.0.3 and 10.244.1.6, a `fwmark` will be internally assigned to the "blue" table and the following will be created on the host:
+The `EgressService` supports a `network` field to specify to which network the egress traffic of the service should be steered to.
+When it is specified the relevant `ovnkube-nodes` take care of creating ip rules on their host.
+Assuming an `EgressService` has `Network: blue`, a ClusterIP of 10.96.135.5 and its endpoints are 10.244.0.3 and 10.244.1.6 the following will be created on the host:
 
 ```none
-$ iptables-save
-*mangle
--A POSTROUTING -j OVN-KUBE-EGRESS-SVC
--A OVN-KUBE-EGRESS-SVC -s 10.244.0.3/32 -m comment --comment default/some-service -j MARK --set-mark 100
--A OVN-KUBE-EGRESS-SVC -s 10.244.1.6/32 -m comment --comment default/some-service -j MARK --set-mark 100
--A OVN-KUBE-EGRESS-SVC -s 10.96.135.5/32 -m comment --comment default/some-service -j MARK --set-mark 100
-
 $ ip rule list
-100:	from all fwmark 100 lookup blue
+5000:	from 10.96.135.5 lookup blue
+5000:	from 10.244.0.3 lookup blue
+5000:	from 10.244.1.6 lookup blue
 ```
 
-This makes the egress traffic of endpoints of an EgressService to be marked before a routing decision is made in order to hit the corresponding ip rule that uses the right routing table.
-A marking rule is also created for the ClusterIP of the service which is needed in order for the return traffic (reply to an external client calling the service) to use the correct table - this is because the packet flow of contacting a LoadBalancer service goes:
+This makes the egress traffic of endpoints of an EgressService to be routed via the "blue" routing table.
+An ip rule is also created for the ClusterIP of the service which is needed in order for the return traffic (reply to an external client calling the service) to use the correct table - this is because the packet flow of contacting a LoadBalancer service goes:
```

Comment:
show an example rule? this part is a bit confusing to me as to how it works.

Maybe also say ingress reply traffic rather than "return traffic".

Reply:
done + the example is above:
```
$ ip rule list
5000:	from 10.96.135.5 lookup blue
```